assignment-04.Rmd

```diff
@@ -45,14 +46,17 @@ a. Look through the help documentation (?CO2) to understand what each variable m
 b. How much does `uptake` change if `conc` goes up by 10 mL/L? (*Note:* it is intentional that there is no mention of the other variables in the model.) Write out the interpretation as a simple statement of this contribution of `conc` on `uptake`, when the other variables are also in the model. (2 marks)
 
 c. Run the following code if you need to download our survey data.
-```{r}
+```{r message=FALSE, warning=FALSE}
 download.file("https://ndownloader.figshare.com/files/2292169", "survey.csv") #if you need to re-download the survey
 survey <- read_csv("survey.csv")
 ```
-    Use logistic regression to see if weight significantly predicts sex. Make a              concluding statement as to indicate whether the model is significant and create a        plot to visualize the linear model. Run the following code to ensure sex is treated      as a factor variable:
+
+Use logistic regression to see if weight significantly predicts sex. Make a concluding statement as to indicate whether the model is significant and create a plot to visualize the linear model. Run the following code to ensure sex is treated as a factor variable:
+
 ```{r}
 survey$sex <- as.factor(survey$sex)
 ```
-      Hint: you need to make sure there are only two levels to this variable: ``"F"`` and       ``"M"``. (0.5 marks)
+
+Hint: you need to make sure there are only two levels to this variable: ``"F"`` and ``"M"``. (0.5 marks)
```

Comment:
Just fyi, you don't need two backticks for inline code highlighting, you only need one.